src/ArbitrumFoundationVestingWallet.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts-upgradeable/finance/VestingWalletUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {IERC20VotesUpgradeable} from "./Util.sol";
+
+interface IL2ArbitrumGoverner {
+    function token() external view returns (IERC20VotesUpgradeable);
+    function EXCLUDE_ADDRESS() external view returns (address);
+}
+
+/**
+ * @notice A wallet for foundation owned founds as per AIP-1.1 specification.
+ * DAO can migrate funds to new wallet.
+ * Wallet vests funds over time on a linear schedule.
+ * Governance votes are delegated to exclude address.
+ * Beneficiary can be updated by DAO.
+ */
+
+contract ArbitrumFoundationVestingWallet is VestingWalletUpgradeable, OwnableUpgradeable {
+    using SafeERC20 for IERC20;
+
+    address private _beneficiary;
+
+    event BeneficiarySet(address newBeneficiary, address caller);
+    event TokenMigrated(address token, uint256 amount, address destination);
+    event EthMigrated(uint256 amount, address destination);
+
+    constructor() {
+        _disableInitializers();
+    }
+    /// @param _beneficiaryAddress Can release funds and receives released funds
+    /// @param _startTimestamp The time to start vesting
+    /// @param _durationSeconds The time period for funds to fully vest
+    /// @param _arbitrumGoverner Core DAO Governer address
+    /// @param _owner Arbitrum DAO. Can migrate funds to new wallet and change beneficiary
+
+    function initialize(
```

Comment:
seems like a blank line here
```suggestion
    /// @param _owner Arbitrum DAO. Can migrate funds to new wallet and change beneficiary
    function initialize(
```

Reply:
the linter was includes it for some reason

Reply:
maybe linter gets happier if you use multiline comment natspec instead? `/**`